README.md

```diff
@@ -75,14 +73,14 @@ The model will be downloaded and cached, and the task will be queued for executi
 Event types:
 - PendingEvent
 - StartedEvent
-- FinishedEvent (with `blob_id`)
+- FinishedEvent (with `image_url` and `parameters_used`)
 - CancelledEvent (with `reason`)
 
 #### Results
 
-A FinishedEvent contains an `image` field including its `blob_id` and `parameters_used`.
-
-`GET /blob/{blob_id}` to get the image in PNG format.
+The FinishedEvent contains a URL to the generated image.
+Currently only local blob serving is supported;
+make sure to download the image before shutting down the server.
```

Comment:
If/when this project adapts to support audio/video, the returned content may not be an image.  It feels like `blob_id` may be best long term, even if for now it is always an image?

Reply:
Hmm, you're right. Perhaps the best course of action is to rename `image_url` => `blob_url`, and `GeneratedImage` => `GeneratedBlob`?

Reply:
Sorry, I meant `blob_url` before.  But yes, I like your suggestion here.